__tests__/app.test.js

```diff
@@ -432,7 +432,37 @@ describe('Test PATCH /api/articles/:article_id endpoint', () => {
         expect(body).toEqual({ msg: "Article Not Found" })
       })
   })
+})
+
 
 
+describe('Test delete /api/comments/:comment_id endpoint', () => {
+  test('Test connection with endpoint, should return 204', () => {
+    return request(app)
+      .delete('/api/comments/1')
+      .expect(204)
+  })
+  test('Test connection with endpoint, should return 204', () => {
+    return request(app)
+      .delete('/api/comments/5')
```

Comment:
What is the difference between this test block and the one above? Both seem to be testing that DELETE works on /api/comments/:comment_id